### PR TITLE
Opt-in styling for tables

### PR DESCRIPTION
**Pull request by mfilej**
Previous rule was too inclusive, styling tables we didn't want styled.

### CODE REVIEW

File: assets/css/rug14.sass [hunk 1]
```diff
@@ -593,4 +588,11 @@ a.pagination-item
   img
     width: 140px
 
+.donations-via-stickers-table
```
Comment (mrfoto) [inline assets/css/rug14.sass]: A bi dal kej bl generičnga? `.with_border` al neki.

Comment (mfilej) [inline assets/css/rug14.sass]: To mam jst pol rajš da se nardi SASS mixin in se ga pol apply-ja okrog, če pride potreba.

Comment (mrfoto) [inline assets/css/rug14.sass]: OK, do that then

Comment (mfilej) [inline assets/css/rug14.sass]: Fixed

**Pull request merged**